cpu/nrf51/radio/nrf51_ble/nrf51_ble.c

```diff
@@ -0,0 +1,628 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_nrf51_ble
+ * @{
+ *
+ * @file
+ * @brief       Implementation of the NRF51822 minimal BLE radio driver
+ *
+ * @author      Jan Wagner <mail@jwagner.eu>
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include "cpu.h"
+#include "mutex.h"
+#include "thread.h"
+#include "sched.h"
+#include "kernel.h"
+#include "periph_conf.h"
+#include "periph/cpuid.h"
+#include "net/gnrc.h"
+#include "hwtimer.h"
+#include "net/ble_ll.h"
+
+#define ENABLE_DEBUG		(0)
+#include "debug.h"
+
+/**
+ * @brief   Driver specific device configuration
+ * @{
+ */
+#define CONF_MODE               (RADIO_MODE_MODE_Ble_1Mbit << RADIO_MODE_MODE_Pos)
+#define CONF_PAYLOAD_LEN        (250U)
+#define CONF_LEN                (8U)
+#define CONF_S0                 (1U)
+#define CONF_S1                 (0U)
+#define CONF_STATLEN            (0U)
+#define CONF_BASE_ADDR_LEN      (3U)
+#define CONF_ENDIAN             RADIO_PCNF1_ENDIAN_Little
+#define CONF_WHITENING          RADIO_PCNF1_WHITEEN_Enabled
```

Comment:
I'm not 100% sure if this is in our coding conventions, but could you just use upper case letters in these macros (lines 40, 47, 48)